Drop net feature

Since the sequoia-net crate is not in debian yet, and tokio is not up
to the version that sq needs, we just build without sequoia-net or tokio.

The compression-related features aren't necessary
(sequoia-openpgp/default already has compression enabled), and the
choice of crypto backends isn't relevant on debian, where we just rely
directly on nettle.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -53,11 +53,6 @@
 optional = true
 default-features = false
 
-[dependencies.sequoia-net]
-version = "0.23"
-optional = true
-default-features = false
-
 [dependencies.sequoia-openpgp]
 version = "1.1"
 default-features = false
@@ -68,10 +63,6 @@
 [dependencies.term_size]
 version = "0.3"
 
-[dependencies.tokio]
-version = "0.2.19"
-features = ["rt-core", "io-util", "io-driver"]
-optional = true
 [dev-dependencies.assert_cli]
 version = "0.6"
 [build-dependencies.clap]
@@ -83,13 +74,7 @@
 
 [features]
 autocrypt = ["sequoia-autocrypt"]
-compression = ["buffered-reader/compression", "sequoia-openpgp/compression"]
-compression-bzip2 = ["buffered-reader/compression-bzip2", "sequoia-openpgp/compression-bzip2"]
-compression-deflate = ["buffered-reader/compression-deflate", "sequoia-openpgp/compression-deflate"]
-crypto-cng = ["sequoia-openpgp/crypto-cng"]
-crypto-nettle = ["sequoia-openpgp/crypto-nettle"]
-default = ["buffered-reader/compression", "sequoia-openpgp/default", "net", "autocrypt"]
-net = ["sequoia-net", "tokio"]
+default = ["buffered-reader/compression", "sequoia-openpgp/default", "autocrypt"]
 [badges.gitlab]
 repository = "sequoia-pgp/sequoia"
 
